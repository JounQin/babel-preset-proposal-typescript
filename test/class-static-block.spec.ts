import { C } from './class-static-block'
import { execute } from './helpers'

const proposal = 'class-static-block'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(C.y).toEqual('bad')
})
