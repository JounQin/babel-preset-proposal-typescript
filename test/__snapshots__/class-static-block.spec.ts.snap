// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class-static-block 1`] = `"Unexpected token ')'"`;
